### ISSUE TITLE
Content Plan Task--Review & update global content

### ISSUE THREAD

**Issue by JuLeeAtPlanet** [planetlabs/planet-client-python#547]
Versioning, date|time stamp, licensing, privacy, security, and global navigation terms and titles should have their own review and update.

**Comment by JuLeeAtPlanet** [comment]
this include nav in docs global upper right 
<img width="262" alt="Screenshot 2023-03-28 at 1 11 17 PM" src="https://user-images.githubusercontent.com/91982271/228355747-836249cc-27e0-4ac5-827e-a362acdeea72.png">

**Comment by jreiberkyle** [comment]
@JuLeeAtPlanet Sara requested that I take this task over. To help me out, can you elaborate on what you'd like to accomplish here?

**Comment by JuLeeAtPlanet** [comment]
This would be a sweep of the corners and boilerplates of the interfaces—an unofficial audit—and update of any values that apply to global interface elements that may not have been addressed since v1.whatever.

So for versioning, you'll note in the docs global upper right, the versioning is 1.5.2 instead of 2.0.0... Also, do we want to have the docs URL to include the -v2? (ie: https://planet-sdk-for-python-v2.readthedocs.io/en/latest/)

For data/time stamps, I think I have the copyright on a routine, so © 2023 [Planet Labs PBC](https://planet.com/) will update to 2024, automatically. But you might want to do a search against the codebase for earlier dates.

For licensing: do we have the appropriate comments in the interface files to declare licensing, indemnification, or other notations that, say, our legal advisors would want us to have? 

For security: have we called out security considerations where appropriate?

Finally, global navigation terms and titles already had their review and update, but a quick sweep to make sure we're not referencing beta or anything in the titles that appear in the table of contents or navs.
